massa-execution-worker/src/module_cache.rs

```diff
@@ -0,0 +1,58 @@
+use massa_execution_exports::ExecutionError;
```

Comment:
Add some doc here to explain how the cache is working and maybe some quick refresher for what is a LRU cache?

Reply:
Done in the `ModuleCache` description